src/app-layout/visual-refresh-toolbar/skeleton/index.tsx

```diff
@@ -111,6 +115,15 @@ export function SkeletonLayout({
       >
         {tools}
       </div>
+      <div
+        className={clsx(
+          styles['global-tools'],
+          !globalToolsOpen && styles['panel-hidden'],
+          sharedStyles['with-motion']
```

Comment:
**Before merge:**

Is this line needed here?

Reply:
no. removed